pkg/live/rgpath.go

```diff
@@ -64,6 +65,11 @@ func (r *ResourceGroupPathManifestReader) Read() ([]*unstructured.Unstructured,
 		if err != nil {
 			return objs, err
 		}
+
+		// Skip if current file is a resourcegroup resource.
```

Comment:
This comment would be much more useful if it explained why.

Reply:
Done. Replaced comment with:

> Skip if current file is a ResourceGroup resource. We do not want to apply/delete any ResourceGroup CRs when we run any `kpt live` commands on a package. Instead, we have specific logic in place for handling ResourceGroups in the live cluster.